cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/SlangModellerImpl.java

```diff
@@ -65,6 +65,7 @@ private void validateFileName(String executableName, ParsedSlang parsedSlang, Ex
         String fileName = parsedSlang.getName();
         if (StringUtils.isNotEmpty(executableName) && !executableName.equals(fileName))
             result.getErrors().add(new IllegalArgumentException("Operation/Flow " + executableName +
-                    " should be declared in a file named \"" + executableName + "." + Extension.SL.getValue() + "\""));
+                    " is declared in a file named \"" + fileName + "." + Extension.SL.getValue() + "\"" +
```

Comment:
appending the `.sl` extension to the name of the SlangSource leads to several problems.. e.g. file.txt -> file.txt.sl - please add test case for this